modal/README.md

```diff
@@ -0,0 +1,62 @@
+# Modal
+
+## Overview
+
+[Modal][1] lets you run generative AI models, large-scale batch jobs, job
```

Comment:
Might be helpful to be specific here about how Modal runs the infrastructure to power these jobs

Reply:
It's feels slightly wordy to be specific here - we make use our custom built containerization tech and filesystems to do so - details we generally don't make apparent to users, hoping it's all seamless. Given we've linked to the modal homepage (where the docs talk through some of these details), what do you think leaving it as is?

Reply:
Gotcha - definitely dont have to do go into details about the custom containerization, but I think it would add context to the idea that "Modal lets you run xyz" just to explain how it does that. Fine leaving as-is if you think this is descriptive enough!

Reply:
Ack, I'll leave it as is, but mostly because I'm finding it hard to phrase it too differently without losing folks.